### DISCUSSION TITLE
Rengoku II - The Stairway To H E A V E N

### DISCUSSION THREAD

**Discussion by AVM-1996** [jserodio/rg351p-ppsspp-settings#16] [New games]
So i just moved my PSP ISO and saw that the game was running fine, i even loaded one of my old saves in the -B floors in the game and i wasn't running frame skip and had frame buffer 0 and the game was running like "hiccups" from time to time BUT! Frame skip kind of did a lot and i even upscaled the 2x res, 2x tex and 2x AA + Super Sampling AA post FX but here is the banana that i really can't stress enough. See when enemy spawn they generate a field around them(like how the terminator time traveled) and this kills the frame rate from 30 to 9 and until i find out how to reduce its effect, running the game with frame skip and no frame buffer is playable with now issue(even fought two bosses to see it myself).

Now if anyone wants to join me in the hell that is **_Dark Souls of Terminators_**, join this discussion and see how if you can kill "_that_" effect!

**Comment by jserodio** [comment]
So basically if I understand right, auto frame skip 1 and no frame buffer is playable without any issues?
